addons/mail/migrations/12.0.1.0/post-migration.py

```diff
@@ -31,9 +31,25 @@ def fill_mail_notification_mail_id(cr):
     )
 
 
+def remove_admin_alias_owner(cr):
+    """Admin's aliases would fail.
+
+    Admin is going to be disabled later in base end-migration.
+    When that happens, any aliases belonging to him would fail.
+
+    By removing the alias owner, we let Odoo decide the default, which will
+    be __system__ usually, the new admin replacement.
+    """
+    openupgrade.logged_query(
+        cr,
+        "UPDATE mail_alias SET alias_user_id = NULL WHERE alias_user_id = 1",
```

Comment:
Without this query, ownership of the aliases would be transferred to the (new) Admin user in the end migration. I see system aliases do not have an owner in new Odoo 12 databases but can have the Admin user in older databases so I think resetting the alias_user_id here is correct.